frontend/src/routes/_dashboard/index.lazy.tsx

```diff
@@ -101,83 +80,129 @@ export function ProjectList() {
   const [currentPage, setCurrentPage] = React.useState(1);
   const [perPage, setPerPage] = React.useState(10);
 
+  const query = useProjectList();
+
+  if (query.isLoading) {
+    return <Loader />;
+  }
+  const projectList = query.data?.data?.results;
+
+  return (
+    <>
+      {projectList.length === 0 ? (
+        <main className="flex gap-3 flex-col items-center justify-center flex-grow h-[75vh]">
+          <div>
+            <h1 className="text-2xl font-bold">Welcome to ZaneOps</h1>
+            <h1 className="text-lg">You don't have any project yet</h1>
+          </div>
+          <Button>Create One</Button>
+        </main>
+      ) : (
+        <main>
+          <div className="md:my-10 my-5">
+            <h1 className="text-3xl  font-bold">Overview</h1>
+            <h4 className="text-sm mt-2 opacity-60">List of projects</h4>
```

Comment:
This should be shown at all times, the list of projects can be in a child component

Reply:
why ? for me this doesn't matter , can you give more details about moving this

Reply:
This is related to the skeleton UI, with the skeleton, the loader will be shown below this title and subtitle.

Reply:
okay but this is not the PR for the skeleton , will take care of that ... noted